Automattic/vip-go-mu-plugins
# Updated Max Queue Settings to 50,000 queue size and 8 hour lag.

## Description

Reducing max Jetpack Settings to account for recent performance updates.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Apply Patch
1. Manually set settings higher than thresholds in Jetpack Debugger
1. Verify settings reset to max limit
1. Manually set settings lower than thresholds in Jetpack Debugger
1. Verify settings reset to min limit

## Follow-up Items

The current tests have settings related to earlier limits. Need to review to get them aligned with final settings that we want.
